src/main/java/reposense/parser/ZoneIdArgumentType.java

```diff
@@ -0,0 +1,29 @@
+package reposense.parser;
+
+import java.time.DateTimeException;
+import java.time.ZoneId;
+
+import net.sourceforge.argparse4j.inf.Argument;
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+import net.sourceforge.argparse4j.inf.ArgumentParserException;
+import net.sourceforge.argparse4j.inf.ArgumentType;
+
+/**
+ * Verifies and parses a string-formatted zone id to a {@code ZoneId} object.
+ */
+public class ZoneIdArgumentType implements ArgumentType<ZoneId> {
+    private static final String MESSAGE_TIMEZONE_INVALID =
+            "The timezone provided is invalid, please use the format of UTC±[hh[mm]]";
+
+    @Override
+    public ZoneId convert(ArgumentParser parser, Argument arg, String value) throws ArgumentParserException {
+        try {
+            if (!value.contains("UTC")) {
+                throw new DateTimeException(MESSAGE_TIMEZONE_INVALID);
```

Comment:
Seems like other timezone other than UTC were still banned.
I agree with @yamidark https://github.com/reposense/RepoSense/pull/496/files#r265014150 that we should allow other instances of timezone, and only mention the more popular ones such as UTC and GMT in our examples.